extensions/extensions-helm/aissemble-spark-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,140 @@
+{{- /*
+aiSSEMBLE Custom rbac.yaml
+
+Required custom rbac.yaml file that grants the sparkoperator service account 
+create, delete, and update access to the apigroup apiextensions.k8s.io. 
+
+This is necessary for the pipeline-invocation-service to create instances of the 
+SparkApplication CRD to submit pipelines to the Spark Operator for execution.
+*/}}
+
+{{- if or .Values.rbac.create .Values.rbac.createClusterRole }}
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I: Primary change for cherry-picked Spark Operator fix (along with docs updates and `rbac` settings in `values.template.yaml`